src/assets/styles/_grid.scss

```diff
@@ -1,11 +1,11 @@
 .gridded {
     position: relative;
     display: grid;
-    margin: 149px 20px;
+    margin: 9.313em 1.25em;
```

Comment:
Something I discovered that Nuxt does recently is static calc functions get computed during server rendering. So `calc(149em / 16)` might be something you want to use here instead of `9.313em`. The client/browser will get the later regardless of what you decide to do.

Reply:
I also noticed that and was wondering whether to keep the math or use the computed values. That number is actually on the typographic scale referenced in our design handbook, though (yet to be released)

Reply:
Ah. In that case we may want to give these specific values variable names in the future.